### ISSUE TITLE
Find a more suggestive name for "API Documentation"

### ISSUE THREAD

**Issue by robert-altom** [robert-altom/test#316]
- API Commands ? 

- we can also move it after the Get Started or AUT Editor section

SEO improvements:
 - Pagetitles should end with: `- AltUnity Tester Documentation` instead of `- AltUnity Tools V.1.6.6 documentation`
the goal is to make the links in google specify AltUnity Tester 

![image](https://gitlab.com/Altom/AltUnity/AltUnityTester/uploads/4d3c5d0fcb06287484164b54ab5519d4/image.png)

---
<sub>You can find the original issue from GitLab [here](https://gitlab.com/altom/altunity/altunitytester/-/issues/592).</sub>

**Comment by robert-altom** [comment]
Tested on the HTML page generated for docs in the cloned project updated from the master branch (for now the alpha-dcos are not 
deployed to check there)

- :heavy_check_mark: Checked that _API Documentation_ section is now named _API_ (as agreed among the AUT team)
- :heavy_check_mark: Checked that the _API_ section is located after the _Get Started_ and _AltUnity Tester Editor_ sections 9as agreed among the AUT team)

  ![image](https://gitlab.com/Altom/AltUnity/AltUnityTester/uploads/06d28b5b923be4b60380a77da6886dd9/image.png)

- :heavy_check_mark: Checked that the page name ends with: `- AltUnity Tester Documentation` instead of `- AltUnity Tools V.1.6.6 documentation`
- :heavy_check_mark: Checked that there are no display issues

The issues are fixed. Closing the ticket.

**Comment by robert-altom** [comment]
API Commands won't work because there are pieces of information that aren't related to commands (ex. BY-Selector).
After a quick search I found that "The API documentation is a reference manual with all needed information about the API, including functions, classes, return types, and arguments.".
I found expressions like: 
- in GitHub docs: REST API
- Bitbucket API
- Java API
- WebSocket API
- Atlassian Python API's Documentation
- API documentation
